galleries/examples/subplots_axes_and_figures/secondary_axis.py

```diff
@@ -40,6 +40,26 @@ def rad2deg(x):
 secax.set_xlabel('angle [rad]')
 plt.show()
 
+# %%
+# Normally, secondary axes are placed at either the 'top' or 'bottom'
```

Comment:
This is not true; you can pass any location. What's the default is that the coordinate space is the Axes.

Reply:
Good point. How about:
> By default, the secondary axis is drawn in the Axes coordinate space.
> We can also provide a custom transform to place it in a different
> coordinate space. Here we put the axis at Y = 0 in data coordinates.

Reply:
That sounds good.